src/i18n/langs/en.json

```diff
@@ -358,11 +359,12 @@
       "send_lightning": "Send funds over Lightning",
       "send_onchain": "Send funds On-chain",
       "unlock": "Unlock",
-      "unlock_subtitle": "Unlock your Wallet",
-      "unlock_success": "Wallet unlocked",
+      "unlock_subtitle": "Unlock your Lightning wallet",
```

Comment:
Not sure if this is used for LN-wallet only or not

Reply:
That's used in the Dashboard for the `UnlockModal`